redfish/data_source_redfish_virtual_media_test.go

```diff
@@ -0,0 +1,42 @@
+package redfish
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
```

Comment:
can we use the library which Akash pointed during the meeting ??

Reply:
As discussed, we will target migrating test files from sdk v2 to plugin framework along with the dev code.